eraser-map.yaml

```diff
@@ -902,6 +902,13 @@ layers:
                 priority: 1
                 sprite: capital-xl
                 size: 18px
+    route_icon:
+        data: { source: route_icon }
+        draw:
+            icons:
+                priority: 1
+                sprite: airport
+                size: 18px
```

Comment:
Can you add `collide: false`, this won't make it disappear like mentioned here https://github.com/mapzen/eraser-map/issues/84#issuecomment-152560078